docs/release/release-team.md

```diff
@@ -89,10 +90,35 @@ When assembling a release team, the release team lead should look for volunteers
 
 ## Time Commitment
 
-As a member of the release team, you should expect to spend approximately 4-8 hours a week on release related activities for the duration of the term. 
+As a member of the release team, you should expect to spend approximately 4-8 hours a week on release related activities for the duration of the term.
+
+Here's a rough estimate of the time requirements per week:
+
+   * Weeks 1-4: 4-8 hours a week
+   * Weeks 5-15: 6-10 hours a week
+   * Week after Code Freeze: Be prepared to dedicate a significant amount of time each day to release-related activities.
+   * Weeks 16-18: 6-10 hours a week
+   * Release Day: Prepare to spend the entire day on release-related activities. If the choice is between working after-hours or on weekends and having the release delayed beyond the initial target, delaying the release is preferred.
```

Comment:
Is this for Team Leads? Shadows or RT Lead itself?

I believe this varies on the team, and the role itself really and we could provide approximate numbers based on the input from sub-team leads to get an idea and cover it for each team separately since teams might have different time commitments/workloads. 
I am planning to propose a doc re-organization in the release folder in the coming weeks (i.e separating out team-specific notes into their own folders similar to how the k8s release team [structured](https://github.com/kubernetes/sig-release/tree/65800cefb8cd72d2cf2fd72e2a5d5c6eaa7b769f/release-team/role-handbooks) it) and include time commits with it, so until then perhaps it is better if we drop this. 

Any other thoughts, @g-gaston @nawazkh

Reply:
Dropped the section.